unix/runner

```diff
@@ -78,19 +78,40 @@ class ResultCallback(CallbackBase):
         self.host_ok = {}
         self.host_unreachable = {}
         self.host_failed = {}
+        self.host_no_mounts = {}
 
     def v2_runner_on_unreachable(self, result):
         host = result._host
         self.host_unreachable[host.get_name()] = result
```

Comment:
What about just printing to STDERR here with the `eprint()` method?

that way errors come out as being processed to STDERR (cool, inline!)

as well as being output at the end in our JSON object to STDOUT

Reply:
Implemented in https://github.com/tidalmigrations/machine_stats/pull/11/commits/319bbc323baa37feb87d9ef11425a8d98bc9b4bf